src/components/NewProject/index.jsx

```diff
@@ -46,6 +47,12 @@ function NewProject() {
     return figmaUrlPattern.test(figmaUrl);
   };
 
+  const getUserInformation = async () => {
+    const { handle: username, img_url: userProfile } = await getUserProfile();
```

Comment:
- `P3`
```suggestion
    const { handle: userName, img_url: userProfile } = await getUserProfile();
```
`userName`이 카멜케이스에 적합하지 않을까 라는 생각입니다!